src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const rows = [...document.querySelectorAll('tr')];
+const column5 = rows.map((row) => {
+  return row.children[4];
+});
+const column2 = rows.map((row) => {
+  return row.children[1].cloneNode(true);
+});
+
+for (let i = 0; i < rows.length; i++) {
+  rows[i].appendChild(column2[i]);
+  rows[i].appendChild(column5[i]);
+}
```

Comment:
Appending `column5[i]` directly to `rows[i]` will remove it from its original position in the DOM. If you intend to keep the original structure intact, consider using `cloneNode(true)` to clone the nodes before appending.